robot/board.py

```diff
@@ -81,15 +81,15 @@ def _socket_with_single_retry(self, handler):
         except retryable_errors:
             raise ConnectionError(self._get_lc_error())
 
-    def receive_raw_from_socket_with_single_retry(self):
+    def _receive_raw_from_socket_with_single_retry(self):
         return self._socket_with_single_retry(
             lambda s: s.recv(Board.RECV_BUFFER_BYTES),
         )
 
     def _recv(self, should_retry=True):
         while b'\n' not in self.data:
             if should_retry:
-                message = self.receive_raw_from_socket_with_single_retry()
+                message = self._receive_raw_from_socket_with_single_retry()
```

Comment:
Apologies, I'd meant inlining the body of the awkwardly named function (rather than moving to a ternary-style statement)